package.json

```diff
@@ -88,10 +88,10 @@
     "yargs": "15.3.1"
   },
   "devDependencies": {
-    "@types/bootstrap": "^4.3.1",
-    "@types/clipboard": "^2.0.1",
+    "@types/find-up": "^2.1.1",
```

Comment:
Is the addition of `@types/find-up` intentional here?

Reply:
Nope, it kinda just slipped in. There were a lot of merge conflicts.